docs/source/faq.rst

```diff
@@ -184,3 +184,19 @@ You can also find the failed trials by checking the trial states as follows:
     trial_id,state,value,...,params,system_attrs
     0,TrialState.FAIL,,...,0,Setting trial status as TrialState.FAIL because of the following error: ValueError('A test error in objective.')
     1,TrialState.COMPLETE,1269,...,1,
+
+
+How to specify the GPU (CUDA) used during execution of a study?
```

Comment:
How about something like this: `How can I use two GPUs for evaluating two trials simultaneously?` I think your current draft explains how to do that. However, I want to make the section title more catchy to help users find this section.

Reply:
Thank you for a useful suggestion.
I adopted your idea and updated the title and example code (https://github.com/pfnet/optuna/pull/263/commits/70eb5d696921b2d600ab342d5eb793b9d783d2ca).